packages/core/src/global_context.rs

```diff
@@ -349,6 +353,9 @@ pub fn schedule_update() -> Arc<dyn Fn() + Send + Sync> {
 /// A component's [`ScopeId`] can be obtained from the [`current_scope_id`] method.
 ///
 /// Note: Unlike [`needs_update`], the function returned by this method will work outside of the dioxus runtime.
+///
+/// Note: The function returned by this method will schedule an update for the component even if it has already updated between when `schedule_update` was called and when the returned function is called.
```

Comment:
I'm not sure this comment is relevant to `schedule_update_any` since it doesn't capture anything about the component. We could still point to `ReactiveContext` as a way to update any reactive scope, not just components, or remove this comment

Reply:
I misread the API. I think the new test is fixed now.